doc/ale-d.txt

```diff
@@ -13,6 +13,27 @@ g:ale_d_dls_executable                                 *g:ale_d_dls_executable*
 See |ale-integrations-local-executables|
 
 
+===============================================================================
+dfmt                                                               *ale-d-dfmt*
+
+g:ale_d_dfmt_executable                               *g:ale_d_dfmt_executable*
+                                                      *b:ale_d_dfmt_executable*
+  Type: |String|
+  Default: `dfmt`
+
+See |ale-integrations-local-executables|
+
+g:ale_d_dfmt_options                                     *g:ale_d_dfmt_options*
```

Comment:
I recommend putting two blank lines between each option.

Reply:
Done in https://github.com/dense-analysis/ale/pull/2712/commits/092d553e671b3d39f5973c2ae8f84ccc878f12dc